org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/AbstractProcessDialog.java

```diff
@@ -478,8 +480,18 @@ protected void savePrameterLayout(HtmlBasedComponent bottomParameterLayout) {
 		bDelete.addActionListener(this);
 		savePrameterLayout.appendChild(bDelete);
 
+		bShare.setTooltiptext(Msg.getMsg(Env.getCtx(), "ShareProcessParameters_XX"));

```

Comment:
just curios, why _XX ? If it is postfix for plugin, should be removed for integration to core.

Reply:
Honestly, I don't remember :) It was probably to draw attention (I'm currently using XXA as prefix).
And, yes it should be removed for core, message should be 'ShareProcessParameters'